Services/Tracking/classes/collection/class.ilLPCollectionOfRepositoryObjects.php

```diff
@@ -431,6 +445,7 @@ public function getTableGUIData(int $a_parent_ref_id): array
             if (count((array) ($grouped_items['items'] ?? [])) > 1) {
                 foreach ($grouped_items['items'] as $grouped_item_id) {
                     if ($grouped_item_id == $item_id ||
+                        !isset($items[$grouped_item_id]) ||
                         !is_array($items[$grouped_item_id] ?? false)) { // #15498
```

Comment:
This line - introduced by akill on 6th of december - fixes the issue. Thus your additional isset(...) is not required anymore